x-pack/plugins/apm/public/components/routing/templates/apm_service_template/index.test.tsx

```diff
@@ -45,12 +45,18 @@ describe('APM service template', () => {
   describe('isInfraTabHidden', () => {
     describe('hides infra tab', () => {
       [
-        { agentName: undefined },
-        { agentName: 'js-base' },
-        { agentName: 'rum-js' },
-        { agentName: 'opentelemetry/webjs' },
-        { serverlessType: ServerlessType.AWS_LAMBDA },
-        { serverlessType: ServerlessType.AZURE_FUNCTIONS },
+        { agentName: undefined, isInfraTabAvailable: false },
```

Comment:
Having `isInfraTabAvailable: true` for these tests should have the same effect right? The logic for showing/hidding infratab is not only dependant on the flag. IMO setting the flag here to `false` is not covering what these tests were covering before. What do you think about leaving `isInfraTabAvailable: true` for these tests and maybe add a couple where the flag is `false`?

Reply:
Good point!